.github/workflows/on-tag.yml

```diff
@@ -21,17 +21,24 @@ jobs:
         with:
           go-version: "1.20.4"
 
-      - name: Build and package resetti
+      - name: Build and package resetti for debian
         run: make deb
+      
+      - name: Build an package resetti from redhat distros
+        run: make rpm
 
       - name: Get release info
         id: tag
         run: echo "::set-output name=tag::$(git describe --tags --abbrev=0)"
 
+      - name: Get rpm file name
+        id: rpm-filename
+        run: echo "::set-output name=filename::$(ls out | grep .rpm)"
+
       - name: Publish release
         uses: ncipollo/release-action@v1
         with:
-          artifacts: "out/resetti.deb,out/resetti,out/bench"
+          artifacts: "out/resetti.deb,out/${{ steps.rpm-filename.outputs.filename }},out/resetti,out/bench"
```

Comment:
Can I inline this? It seems like I can but I wanted to make sure

Reply:
What do you mean here by inline exactly?

Reply:
I fixed it nevermind